docs/build-notifications.rst

```diff
@@ -20,31 +20,256 @@ Take these steps to enable build notifications using email:
 
 You should now get notified by email when your builds fail!
 
-Using webhook
--------------
+Webhooks
+--------
```

Comment:
Should we clarify this a bit more, since the content is scoped to only this?

```suggestion
Build Status Webhooks
------------------------------
```